src/agents/customer_group.jl

```diff
@@ -345,47 +361,47 @@ function CustomerGroup(input_filename::AbstractString, model_data::HEMData; id =
         ParamArray(
             "Constant",
             (model_data.index_h,),
-            [0.0, 0.0, 0.0]; 
+            fill(0.0, length(model_data.index_h)),
             description = "Constant in green power uptake function (regression parameter)",
         ),
         ParamArray(
             "GreenPowerPrice_coefficient",
             (model_data.index_h,),
-            [0.0, -0.55, -0.55];
+            vcat(fill(0.0, length(model_data.index_h) - 2), [-0.55, -0.55]),
```

Comment:
This doesn't work if model commercial or industrial at county-level. Need to use the h_to_sector map. Same below for non-zero values.